SVS/bin/infer.py

```diff
@@ -48,16 +51,28 @@
         help="Combine multiple datasets & add singer embedding",
     )
     parser.add_argument(
-        "--random_crop",
```

Comment:
why remove random crop here

Reply:
There is no need for random_crop during the test period. (No augmentation for test set !